### PR TITLE
meta(gh): Add a status check for changelogs

### PR DESCRIPTION
**Pull request by untitaker**
#skip-changelog

### CODE REVIEW

File: dangerfile.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    return;
+  }
+
+  const hasChangelog = danger.git.modified_files.indexOf('CHANGELOG.md') !== -1;
+  const hasPyChangelog = danger.git.modified_files.indexOf('py/CHANGELOG.md') !== -1;
+
+  const isTrivial = (danger.github.pr.body + danger.github.pr.title).includes('#trivial');
+
+  if (!hasChangelog !hasPyChangelog && !isTrivial) {
+    warn('Please add a changelog entry in either CHANGELOG.md or py/CHANGELOG.md, or add #trivial to the PR title.');
```
Comment (jan-auer) [inline dangerfile.ts]: Instead of the title, can we make it a comment or the PR description?

Comment (untitaker) [reply dangerfile.ts]: I copypasted this from the javascript repo and thought i'd add an instructional msg -- turns out it does use both title and body

**Comment by untitaker** [comment]
@jan-auer tried out github actions over hackweek. it's better. should I move this dangerjs stuff into gha as well?

**Comment by jan-auer** [comment]
@untitaker yes, i think that would make a lot of sense. 

Do you see a way to get this working for external contributions as well? Ideally, this works on all PRs similar to all those license bots.

**Comment by jan-auer** [comment]
Update: You can just use the github action integration documented here:
https://danger.systems/js/guides/getting_started.html#setting-up-danger-to-run-on-your-ci

`secrets.GITHUB_TOKEN` will automatically be a token scoped to this repo (no need to create one). That way it also works for external PRs.

**Comment by untitaker** [comment]
So it seems that GHA is not really running here, not sure what to do

**Review by untitaker — COMMENTED** [review]
i approve

**Review by jan-auer — APPROVED** [review]

**Pull request merged**